docker-compose.devel.yml

```diff
@@ -0,0 +1,38 @@
+version: '3'
+
+services:
+  database:
+    extends:
+      file: docker-compose.yml
+      service: database
+    volumes:
+      - ./database:/git
+    security_opt:
+      - label=disable
+    working_dir: /git
+
+  reposcan:
+    extends:
```

Comment:
'extends' isn't supported by docker v3 :(

https://docs.docker.com/compose/compose-file/compose-versioning/#version-3

Reply:
Oh My Docker... I've naively tested it on my v2 compose :(